tigerpath/templates/tigerpath/privacy.html

```diff
@@ -35,24 +35,30 @@ <h1 class="mb-3">Privacy Policy</h1>
-          <h6>Authentication</h6>
-          <p>We protect the entire application underneath CAS authentication, which is the same system you use to sign into other Princeton services, like TigerHub.</p>
-          <h6>User Data</h6>
-          <p>We are committed to protecting your privacy. Your data is stored securely, and we will never sell your data to third parties. We only store the data that we need to in order to provide you with the best experience possible in TigerPath. This data includes your nickname, class year, major, and four-year course schedule. By using TigerPath, you are consenting to the storage of this data.</p>
-          <h6>University Data</h6>
-          <p>We collect the course listings data from the OIT Web Feed Registry and the Course Offerings webpage, and we compile the requirements information from the Princeton University Undergraduate Announcement website and the official Princeton University department websites.</p>
-          <h6>Third-Party Services</h6>
-          <p>We are using some third-party services in order to provide you with the best experience possible:<p>
-          <ul>
-            <li>TigerBook API, to automatically populate your name, class year, and major.</li>
-            <li>TigerApps Transcript API, to help you prepopulate your course schedule with courses you've already taken.</li>
-            <li>Google Analytics, to collect aggregated anonymous metadata on how you use the app so we can continually improve it.</li>
-          </ul>
-        </div>
+    <section class="container mb-4 mt-4">
+      <div class="text-section">
+        <p>This privacy policy outlines what data we collect about you (the user), where the course offerings and requirements data in TigerPath comes from, and what other third-party services we are using.</p>
+      </div>
+      <div class="text-section">
+        <h4>Authentication</h4>
+        <p>We protect the entire application underneath CAS authentication, which is the same system you use to sign into other Princeton services, like TigerHub.</p>
+      </div>
+      <div class="text-section">
+        <h4>User Data</h4>
+        <p>We are committed to protecting your privacy. Your data is stored securely, and we will never sell your data to third parties. We only store the data that we need to in order to provide you with the best experience possible in TigerPath. This data includes your nickname, class year, major, and four-year course schedule. By using TigerPath, you are consenting to the storage of this data.</p>
+      </div>
+      <div class="text-section">
+        <h4>University Data</h4>
+        <p>We collect the course listings data from the OIT Web Feed Registry and the Course Offerings webpage, and we compile the requirements information from the Princeton University Undergraduate Announcement website and the official Princeton University department websites.</p>
```

Comment:
add:
_While we aim to make the displayed information as accurate as possible, occasionally, errors do occur and requirements sometimes change before we are able to update them. Therefore we make no guarantees about its accuracy. The final [word] about requirements will always rest with the respective department and the university administration._

Possibly also direct them to give feedback if they notice an error.

Reply:
It seems strange to me to have this in the privacy policy. This is something that would fit better in a terms of service

Reply:
Maybe we should just rename the page to something like "Data and Privacy" then.
This page already has stuff that deals with data issues that don't have much to do with privacy, like how we scrape the course data and get the requirements data. I feel like this disclaimer should come attached to it.

I agree that we might want them to eventually be on separate pages.